pipelines/SpeciesTreeFromBusco/main.nf

```diff
@@ -52,7 +52,10 @@ process prepareBusco {
 }
 
 process buscoAnnot {
-    label 'rc_64Gb'
+    label 'lsf_32Gb_16C'
+    cpus 16
+    memory '32 GB'
```

Comment:
Keen to have these removed from the commit, even if you keep them locally temporarily